### ISSUE TITLE
Broken link in documentation

### ISSUE THREAD

**Issue by lslebodn** [lightblue-platform/lightblue-docs-overview#15]
The readme.md[1] contains various external links but DNS records does not exists.
e.g.
Overview -> http://lightblue.io/
Specification -> http://jewzaam.gitbooks.io/lightblue-specifications/content/
User Guide-> http://docs.lightblue.io/
Developer Manula -> http://dev.docs.lightblue.io/

```
sh-5.0$ nslookup lightblue.io 8.8.8.8
Server:         8.8.8.8
Address:        8.8.8.8#53

Non-authoritative answer:
Name:   lightblue.io
Address: 209.132.183.105

sh-5.0$ 
sh-5.0$ nslookup jewzaam.gitbooks.io 8.8.8.8
Server:         8.8.8.8
Address:        8.8.8.8#53

** server can't find jewzaam.gitbooks.io: NXDOMAIN

sh-5.0$ nslookup docs.lightblue.io 8.8.8.8
Server:         8.8.8.8
Address:        8.8.8.8#53

** server can't find docs.lightblue.io: NXDOMAIN

sh-5.0$ nslookup dev.docs.lightblue.io 8.8.8.8
Server:         8.8.8.8
Address:        8.8.8.8#53

** server can't find dev.docs.lightblue.io: NXDOMAIN
```

[1] https://github.com/lightblue-platform/lightblue-docs-overview/blob/master/README.md

**Comment by lslebodn** [comment]
@jewzaam Are you responsible for jewzaam.gitbooks.io ?

**Comment by davidlanouette** [comment]
tldr; this looks like a dns issue.

&nbsp;

Using `dig`, it looks like most of these urls (docs.lightblue.io, jewzaam.gitbooks.io and dev.docs.lightblue.io) all eventually resolve to www.gitbooks.io.

```
$ dig dev.docs.lightblue.io 8.8.8.8
...
;; ANSWER SECTION:
dev.docs.lightblue.io.	300	IN	CNAME	www.gitbooks.io.
www.gitbooks.io.	1584	IN	CNAME	cdn.gitbook.com.
```

However, "gitbooks.io" seems like a broken dns entry.  `dig` returns no answer section for that domain.

NOTE: www.lightblue.io also resolves to www.gitbooks.io.  The default subdomain appears to be redirecting to the www subdomain.

```
$ wget http://lightblue.io
--2020-06-04 08:59:01--  http://lightblue.io/
Resolving lightblue.io (lightblue.io)... 209.132.183.105
Connecting to lightblue.io (lightblue.io)|209.132.183.105|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: http://www.lightblue.io [following]
--2020-06-04 08:59:01--  http://www.lightblue.io/
Resolving www.lightblue.io (www.lightblue.io)... failed: nodename nor servname provided, or not known.
wget: unable to resolve host address ‘www.lightblue.io’
```

**Comment by davidlanouette** [comment]
After more digging, it looks like we need to update the CNAME to `hosting.gitbook.com`

According to the "DNS Settings" section of the [custom domains](http://docs.gitbook.com/hosting/custom-domains#dns-settings) page, the `www.gitbooks.io` name is for legacy GitBooks, and may be deprecated.  Maybe they finally pulled the pin on that deprecation.

> **DNS Settings**
> In short, CNAME to `hosting.gitbook.com`.
> 
> If you are a v1 user, you may have been used to using `www.gitbooks.io`, which only works for legacy.gitbook.com and will not work for new users.
> 
> ​**Note**: GitBook legacy will soon be deprecated. Contact us to help you migrate your content.
> 

​

**Comment by jewzaam** [comment]
Wow, I haven't managed this for many years now.  I could get DNS setup but haven't looked at any of the gitbook stuff for a while, looks like it changed a lot.  Anybody interested in getting docs building setup and let me know what has to be done for DNS?

**Comment by jewzaam** [comment]
FYI @bserdar

**Comment by davidlanouette** [comment]
It looks like the DNS for this server is controlled by Red Hat IT.  I'm working to get that fixed now.

**Comment by lslebodn** [comment]
works now
